docs/sections/overview/usage.md

```diff
@@ -0,0 +1,205 @@
+`content_header` | **main** | To fill the header element of the page (will be placed above the main content)
+`content` | **main** | To fill all of the main content of the page
+`footer` | **main** | To fill the content of the footer section
+`right_sidebar` | **main** | To fill the content of the right sidebar (or control sidebar)
+`css` | **main** | To add extra style sheets (inside the `<head>` tag)
+`js` | **main** | To add extra scripts or javascript code (just before the closing `</body>` tag)
+`adminlte_css_pre` | misc | To add custom style sheets before the style sheets required by AdminLTE
+`content_top_nav_left` | misc | To add custom elements in the left section of the top navbar.
+`content_top_nav_right` | misc | To add custom elements in the right section of the top navbar.
+`meta_tags` | misc | To add extra meta tags inside the `<head>` tag
+`preloader` | misc | To allow the replacement of the preloader animation default content. Requires package version `v3.9.5` or greater and the preloader animation enabled by configuration
+`usermenu_header` | misc | To allow the replacement of the header in the usermenu dropdown by a custom version. Requires an authentication scaffolding and the usermenu enabled by configuration
+`usermenu_body` | misc | To add a custom body element into the usermenu dropdown. Requires an authentication scaffolding and the usermenu enabled by configuration
+
+> [!IMPORTANT]
+> The `right_sidebar` section was named `right-sidebar` before version `v3.14.0`. So be sure to use the correct name depending on the package version you're using.
+
+All the previously described sections are optional. As a basic example, your most common blade file extending the provided template could look like the following one:
+
+```blade
+@extends('adminlte::page')
+
+@section('title', 'Dashboard')
+
+@section('content_header')
+    <h1>Dashboard</h1>
+@stop
+
+@section('content')
+    <p>Welcome to this beautiful admin panel.</p>
+@stop
+
+@section('css')
+    {{-- Add here extra stylesheets --}}
+    {{-- <link rel="stylesheet" href="/css/admin_custom.css"> --}}
+@stop
+
+@section('js')
+    <script> console.log("Hi, I'm using the Laravel-AdminLTE package!"); </script>
+@stop
+```
+
+> [!Tip]
+> On a fresh **Laravel** installation, and after installing this package, you can replace the `resources/views/welcome.blade.php` file with the previous code for a fast template review.
+
+Now, and as usual, you just return this view from a controller. It's a recommendation to check out [AdminLTE v3](https://adminlte.io/themes/v3/) to find out how to build beautiful content for your admin panel. As a preview, the next image shows what you can get with the previous blade file definition:
+
+> [!Note]
+> Screenshot was taken from package version `v3.9.4`.
+
+![laravel-adminlte-layout-example](/imgs/overview/usage/laravel-adminlte-layout-example.png)
```

Comment:
@gpibarra Instead of having all the images versioned in the repo, do you think is it a good move to use the images already stored in the **Github cloud**? For example, here the Wiki page is using the next image already in Github cloud:

https://github.com/jeroennoten/Laravel-AdminLTE/assets/63609705/5ecb6912-c168-4ed5-be29-dc1e48f5372b

Reply:
When I cloned the wiki, the references to the images were on GitHub cloud, but when publishing with Vitepress some links were broken, so I downloaded them manually to include them in the repo.

In my opinion, having them in the repo is more flexible for anyone who wants to contribute to the documentation and add new images, otherwise I think only the owner of the repo could upload images, right?

Reply:
No, everyone can upload images, but the method is somehow tricky, you need to open a new issue somewhere and attach/upload the image there, then copy the link generated for that image and cancel the issue you were opening.

Reply:
I think it's better to have them in the repo, but let me know if you want me to change it then.

Reply:
OK, thanks for your feedback, I will think about pros and cons to take a decision.